roles/ealapps/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+# tasks file for ealapps
+- name: ealapps | install ealapps dependencies
+  ansible.builtin.apt:
+    name: ["php{{ php_version }}-curl"]
+    state: present
+
+- name: ealapps | install github deploy keys
+  ansible.builtin.template:
+    src: "{{ item }}"
+    dest: "/home/{{ deploy_user }}/.ssh/{{ item }}"
+    owner: "{{ deploy_user }}"
+    group: "{{ deploy_user }}"
+    mode: 0600
+  with_items: "{{ deploy_keys }}"
```

Comment:
Where is the deploy_keys variable defined?

Reply:
https://github.com/pulibrary/princeton_ansible/blob/eaf7664c9cb9d2cd3cac805498f4a203274ae744/roles/deploy_user/defaults/main.yml#L3